### PR TITLE
Updated user doc for onboarding SDK

### PR DESCRIPTION
**Pull request by mikelam-us-aixplain**
Updated Docker login and `create image-repo` commands in documentation.

### CODE REVIEW

File: docs/user/user_doc.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ $ aixplain get image-repo-login [--api-key <TEAM_API_KEY>]
```
Comment (thiago-aixplain) [inline docs/user/user_doc.md:124]: You may want to mention here that the image needs to be built following the standard bellow:

```
docker build . -t 535945872701.dkr.ecr.us-east-1.amazonaws.com/<repositoryName>:<tag>
```

File: docs/user/user_doc.md
Comment (thiago-aixplain) [inline docs/user/user_doc.md:1]: Make sure you explain how to obtain the image hash before onboarding the model:

```
docker images --digests
```

**Review by thiago-aixplain — APPROVED** [review]

File: docs/user/user_doc.md (hunk 1 above)
Comment (mikelam-us-aixplain) [reply docs/user/user_doc.md:124]: Fixed!

File: docs/user/user_doc.md
Comment (mikelam-us-aixplain) [reply docs/user/user_doc.md:1]: Fixed!

**Pull request merged**